Anlab.Mvc/Views/Templates/_OrderCreated.cshtml

```diff
@@ -156,7 +156,7 @@
                                                                                                     <th style="Margin:0;color:#0a0a0a;font-family:Helvetica,Arial,sans-serif;font-size:16px;font-weight:400;line-height:1.3;margin:0;padding:0;text-align:left">
                                                                                                         <p style="Margin:0;Margin-bottom:10px;color:#0a0a0a;font-family:Helvetica,Arial,sans-serif;font-size:16px;font-weight:400;line-height:1.3;margin:0;margin-bottom:10px;padding:0;text-align:left">
                                                                                                             <strong>Created By</strong><br />@createdBy<br /><br /><strong>Sample Type</strong><br />@order.SampleType<br /><br />@if (!String.IsNullOrWhiteSpace(order.Commodity))
-                                                                                                            {<strong>Commodity</strong><br />@order.Commodity<br /><br />} <strong># of Samples</strong><br />@order.Quantity<br /><br />@if (order.DateSampled != null)
+                                                                                                            {<strong>Commodity</strong><br />@order.Commodity<br /><br />} <strong># of Samples</strong><br />@order.Quantity<br /><br />@if (order.DateSampled != default)

```

Comment:
@sprucely Why this change? And the Templates folder I think is just used for email generation. I don't know that it is a good idea to change anything in here....

Reply:
I think cshtml no longer allows comparing a value type to null. It won't build without this change.